matsim/src/test/java/org/matsim/core/router/FallbackRoutingModuleTest.java

```diff
@@ -0,0 +1,93 @@
+package org.matsim.core.router;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.matsim.api.core.v01.Coord;
+import org.matsim.api.core.v01.Id;
+import org.matsim.api.core.v01.Scenario;
+import org.matsim.api.core.v01.network.Link;
+import org.matsim.api.core.v01.network.Network;
+import org.matsim.api.core.v01.network.NetworkFactory;
+import org.matsim.api.core.v01.network.Node;
+import org.matsim.api.core.v01.population.*;
+import org.matsim.core.config.Config;
+import org.matsim.core.config.ConfigUtils;
+import org.matsim.core.config.groups.StrategyConfigGroup;
+import org.matsim.core.controler.AbstractModule;
+import org.matsim.core.controler.Controler;
+import org.matsim.core.replanning.strategies.DefaultPlanStrategiesModule;
+import org.matsim.core.scenario.ScenarioUtils;
+import org.matsim.facilities.Facility;
+import org.matsim.testcases.MatsimTestUtils;
+
+import java.awt.*;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class FallbackRoutingModuleTest{
+	@Rule public MatsimTestUtils utils = new MatsimTestUtils() ;
+
+	@Test
+	public void calcRoute(){
+
+		Config config = ConfigUtils.createConfig() ;
+		config.controler().setOutputDirectory( utils.getOutputDirectory() );
+		config.controler().setLastIteration( 1 );
+
+		StrategyConfigGroup.StrategySettings sets = new StrategyConfigGroup.StrategySettings(  ) ;
+		sets.setStrategyName( DefaultPlanStrategiesModule.DefaultStrategy.ReRoute ) ;
+		sets.setWeight( 1. ) ;
+		config.strategy().addStrategySettings( sets );
+
+		Scenario scenario = ScenarioUtils.createScenario( config ) ;
+
+		Network network = scenario.getNetwork() ;
+		NetworkFactory nf = network.getFactory();
+
```

Comment:
I would have expected consistent code style in a test class which I assume would have been written by a single person. But about 50% of the lines end with a semicolon with a space in fron of it, and the other 50% of the lines do not have a space before. Some of the method arguments have spaces around the parentheses like above (`( config )`), some not (like below: `new Coord(0.,10.)`). Sometimes there are spaces after (and even before) commas separating parameters, sometimes not at all (both cases can be seen on the next line.

I would really prefer if at least within one file the formatting would be consistent, as it makes reading code much more easier.

Reply:
I guess that my manually typed coding style and the intellij autoformatting are not consistent.  I have now intellij-autoformatted the test class; hoping that that will make it better.